components/kyverno/development/kyverno-helm-values.yaml

```diff
@@ -0,0 +1,93 @@
+---
+fullnameOverride: konflux-kyverno
+namespaceOverride: konflux-kyverno
+admissionController:
+  replicas: 1
+  initContainer:
+    securityContext:
+      allowPrivilegeEscalation: false
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+      capabilities:
+        drop:
+        - "ALL"
```

Comment:
Will this work on openshift?  In the past, we've had to set these to `null` [on the recommendation of upstream](https://kyverno.io/docs/installation/platform-notes/#notes-for-openshift-users) so that openshift can set permissions properly.

Reply:
if we set this to null the linter complains. We already have the same configuration in [other deployments](https://github.com/redhat-appstudio/infra-deployments/blob/9ed44289d5355e288c0bb62269ca57d7213f3887/components/namespace-lister/base/deployment.yaml#L54-L60) and it's working